src/dvc_render/vega_templates.py

```diff
@@ -488,7 +488,7 @@ class SmoothLinearTemplate(Template):
             {
                 "loess": Template.anchor("y"),
                 "on": Template.anchor("x"),
-                "groupby": ["rev"],
+                "groupby": ["rev", "filename"],
```

Comment:
Comment: here is the main update that fixes the issue. Everything else is an update to the style to make it the same as the default `linear` plot.